src/components/WriteBarIcon/WriteBarIcon.css

```
@@ -30,30 +30,25 @@
   position: absolute;
   top: 6px;
   right: 2px;
-  box-shadow: 0 0 0 2px var(--background_content);
+  box-shadow: 0 0 0 2px
+    var(--background_content, var(--vkui--color_background_modal));
 }
 
 .WriteBarIcon.WriteBarIcon--send,
 .WriteBarIcon.WriteBarIcon--done {
-  color: var(--accent);
+  color: var(--accent, var(--vkui--color_icon_accent));
 }
 
 /**
  * iOS
  */
 .WriteBarIcon--ios {
-  color: var(--writebar_icon);
-  border-radius: 10px;
+  color: var(--writebar_icon, var(--vkui--color_icon_accent));
```

```suggestion
  color: var(--writebar_icon, var(--vkui--color_icon_accent_themed));
```

нужна `_themed`, чтобы в тёмной теме переключалась в белый цвет – `--writebar_icon` именно так работает

---

> нужна `_themed`, чтобы в тёмной теме переключалась в белый цвет – `--writebar_icon` именно так работает

На макетах вроде бы токен `--vkui--color_icon_accent` 🤔

---

Леша подтвердил, что просто `accent` должно быть